.github/workflows/cve-weekly.yaml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2024 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Weekly CVE Scan
+
+env:
+  MODULES_REGISTRY: ${{ vars.DEV_REGISTRY }}
+  CI_COMMIT_REF_NAME: ${{ github.ref_name }}
+  MODULES_MODULE_NAME: ${{ vars.MODULE_NAME }}
+  MODULES_MODULE_SOURCE: ${{ vars.DEV_MODULE_SOURCE }}
+  MODULES_REGISTRY_LOGIN: ${{ vars.DEV_MODULES_REGISTRY_LOGIN }}
+  MODULES_REGISTRY_PASSWORD: ${{ secrets.DEV_MODULES_REGISTRY_PASSWORD }}
+  WERF_DRY_RUN: "false"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, edited, synchronize]
+  # push:
+  #   branches:
+  #     - main
+  #     - refactor/ci/update-go-add-trivy
+  schedule:
+    - cron: "23 0 * * 6"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  trivy_scan:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set vars
+        run: |
+          echo "REPORT_FILE_NAME=$(date +%Y-%m-%d)-report.txt" >> "$GITHUB_ENV"
+
+      - name: Install Task
+        uses: arduino/setup-task@v2
+        with:
+          version: 3.37.2
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
There is no `github.event.pull_request` for workflow_dispatch and schedule triggers.

Reply:
Just for the test